### PR TITLE
feat: add pytorch to cookbook

### PR DESCRIPTION
**Pull request by bwanglzu**

### CODE REVIEW

File: .github/2.0/cookbooks/Executor.md [hunk 1]
```diff
@@ -514,5 +514,34 @@ class PaddleMwuExecutor(Executor):
...
+    @requests
+    def encode(self, docs, **kwargs):
+        for doc in docs:
+            input_tensor = torch.from_numpy(
+                doc.blob
+            )  # convert the ``ndarray`` of the doc to ``Tensor``
+            output_tensor = torch.matmul(
+                self.encoding_mat, input_tensor
+            )  # multiply the input with the encoding matrix.
+            doc.embedding = (
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:542]: maybe u can flatten to make it one liner?

File: .github/2.0/cookbooks/Executor.md [hunk 2]
```diff
@@ -514,5 +514,34 @@ class PaddleMwuExecutor(Executor):
...
+import numpy as np
+
+
+from jina import Executor, requests
+
+
+class PytorchMwuExecutor(Executor):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.dims = 5
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:530]: u can remove this line and add it in the encoding_mat?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: .github/2.0/cookbooks/Executor.md (hunk 2 above)
Comment (bwanglzu) [reply .github/2.0/cookbooks/Executor.md:530]: I wanna keep it consistent with other cookbooks..

**Comment by bwanglzu** [comment]
please review #2474 first, then I'll resolve the conflict

File: .github/2.0/cookbooks/Executor.md [hunk 3]
```diff
@@ -39,7 +39,14 @@ Table of Contents
     - [Metas](#metas)
     - [`.metas` & `.runtime_args`](#metas--runtime_args)
 - [Migration in Practice](#migration-in-practice)
+<<<<<<< HEAD
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:42]: big mess up

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**